content/guides/getting-started.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Getting Started
+sort: 2
+contributors:
+  - bebraw
+  - varunjayaraman
+  - cntanglijun
+  - chrisVillanueva
+  - johnstew
+  - simon04
+  - aaronang
+  - jecoopr
+---
+
+As you may already know, webpack is used to compile JavaScript modules. Once [installed](/guides/installation), you can interface with webpack either from its [CLI](/api/cli) or [API](/api/node). If you're still new to webpack, please read through the [core concepts](/concepts) and [this comparison](/guides/comparison) to learn why you might use it over the other tools that are out in the community.
+
+
+## Basic Setup
+
+First let's create a directory, initialize NPM, and [install webpack locally](/guides/installation#local-installation):
+
+``` bash
+mkdir webpack-demo && cd webpack-demo
+npm init -y
+npm install --save-dev webpack
+```
+
+Now we'll create the following directory structure and contents:
+
+``` diff
+webpack-demo
+|- package.json
++ |- index.html
++ |- /src
++   |- index.js
+```
+
+__src/index.js__
+
+```javascript
+function component () {
+  var element = document.createElement('div');
+
+  /* lodash is required for the next line to work */
```

Comment:
`/* lodash is required for the next line to work, we will add lodash to the index.html file for now */`
Or something along those lines, so people know where lodash is "coming from"?